pkgs/top-level/aliases.nix

```diff
@@ -782,6 +782,7 @@ mapAliases ({
   pgtap = postgresqlPackages.pgtap;
   plv8 = postgresqlPackages.plv8;
   timescaledb = postgresqlPackages.timescaledb;
+  tlauncher = throw "tlauncher has been removed because there questionable practises around illegal versions of Minecraft";
```

Comment:
I would consider leaving specifics out. How about "tlauncher has been removed because there questionable practices and legality concerns"